### PR TITLE
Chapter 5: Filling in layout

### PR DESCRIPTION
**Pull request by ducbuianh**
### Title
Source for Chapter 5 in "Ruby on Rails Tutorial (Rails 5)"
Link: https://www.railstutorial.org/book/filling_in_the_layout

### Background
Training Rails

### Technical Changes
1. Filling the layout for pages using Bootstrap scss
2. Add new Contact Page
3. Add new SignUp Page

### Test
Run "rails t". All Passed.

### Database Migration (Optional)
N/A #

### CODE REVIEW

File: app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<footer class="footer">
+  <small>
+    The <a href="http://www.railstutorial.org/">Ruby on Rails Tutorial</a>
```
Comment (haanhduclinh) [inline app/views/layouts/_footer.html.erb]: [Q]
Could you please use I18.n to text?
Currently, Our system support multi-language. This is good change to practice that

File: app/assets/javascripts/users.coffee [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (haanhduclinh) [inline app/assets/javascripts/users.coffee]: [Q]
why we need ` app/assets/images/kitten.jpg` ?

File: app/assets/stylesheets/custom.scss [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  text-align: center;
+}
+
+h2 {
+  font-size: 1.2em;
+  letter-spacing: -1px;
+  margin-bottom: 30px;
+  text-align: center;
+  font-weight: normal;
+  color: $gray-light;
```
Comment (haanhduclinh) [inline app/assets/stylesheets/custom.scss]: [Q] where do you define `$gray-light` ?

Comment (ducbuianh) [reply app/assets/stylesheets/custom.scss]: [A] It is supported by bootstrap-sass gem.

File: app/assets/javascripts/users.coffee (hunk 2 above)
Comment (ducbuianh) [reply app/assets/javascripts/users.coffee]: [A] I will delete that. It was a part of the tutorial but unnecessary now.

File: app/views/layouts/_footer.html.erb (hunk 1 above)
Comment (ducbuianh) [reply app/views/layouts/_footer.html.erb]: [A] OK I'll try.

**Pull request merged**